content/200-concepts/150-data-platform/08-billing/02-plans-and-quotas.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: 'Plans and quotas'
+metaTitle: 'Plans and quotas'
+metaDescription: 'The Prisma Data Platform provides a set of monthly plans that include quotas for the number of collaborators and environments per project, as well as a quota for the usage of the Prisma Data Proxy (in CPU milliseconds).'
+---
+
+<TopBlock>
+
+The Prisma Data Platform provides a set of monthly plans that include quotas for:
+
+- number of collaborators per project
+- number of environments per project
+- Prisma Data Proxy usage (in CPU milliseconds) per project
+
+</TopBlock>
+
+## About plans and quotas
+
+Plans in the Prisma Data Platform are designed with the goal to help you scale your projects as the need for team members, environments, and Data Proxy usage grows.
+
+Because of this, a single Prisma Data Platform plan applies for a single project.
+
+We want to give you the option to create projects freely and you can then decide which project you want to upgrade to a paid plan. You can have many Prisma projects and you can upgrade only several of them to a paid plan.
```

Comment:
```suggestion
We want to give you the option to create projects freely and you can then decide which project you want to upgrade to a paid plan. You can have many Prisma projects and you can decide to upgrade only several of them to a paid plan.
```
That makes it clear it's not due to a restriction of the Product, but solely based on the user's decision

Reply:
Agreed. It's now added.